### ISSUE TITLE
no-bitwise doesn't catch the shorthand

### ISSUE THREAD

**Issue by mzgol** [eslint/eslint#1643]
if `no-bitwise` is enabled, the following is considered an error:
```js
var n = 1;
console.log(n >> 1);
```
but the following is not marked as such:
```js
var n = 1;
n >>= 1;
```

This may apply to other operators as well.

ESLint 0.11.0.

**Comment by nzakas** [comment]
Yup, that definitely looks like a bug.
